modules/monitoring-accessing-alerting-rules-for-all-projects.adoc

```diff
@@ -1,11 +1,11 @@
 // Module included in the following assemblies:
 //
-// * monitoring/monitoring-your-own-services.adoc
+// * monitoring/managing-alerts.adoc
 
-[id="accessing-alerting-rules-for-all-namespaces_{context}"]
-= Accessing alerting rules for all namespaces
+[id="accessing-alerting-rules-for-all-projects_{context}"]
+= Accessing alerting rules for all projects
 
-As a cluster administrator, you can access alerting rules from all namespaces together in a single view.
+As a cluster administrator, you can list alerting rules for core {product-title} and user-defined projects together in a single view.
 
 [NOTE]
 ====
```

Comment:
May be fine here, but good to be wary of any "future release" promises.

Reply:
Good point. I have taken the note out because it doesn't really add anything. If we take it out in the future, we can document it then.